cluster/src/test/java/com/hartwig/pipeline/smoke/SmokeTest.java

```diff
@@ -39,4 +130,15 @@ private static String workingDir() {
     private static String noDots(final String version) {
         return version.replace(".", "");
     }
+
+    private static String setName(Arguments arguments) throws IOException {
+        List<SbpSet> sets =
+                ObjectMappers.get().readValue(SbpRestApi.newInstance(arguments).getSet(SET_ID), new TypeReference<List<SbpSet>>() {
```

Comment:
probably better to pass a single instance of the rest api in to these two methods than creating two separate connections.